src/Schema/ColumnSchemaInterface.php

```diff
@@ -86,6 +86,8 @@ public function dbType(string|null $value): void;
      * directly in your code.
      *
      * If the value is `null` or an {@see Expression}, there will be no conversion.
+     * In exceptional case (with direct use) you can use array containing the value as the first element and the PDO type as the second
```

Comment:
After this fix for array `['key' => 0, 1 => DataType::STRING]` returns array, but for array `[0, 2]`  returns `Param` object. It's unexpectedly. May be write in phpdoc "do not use directly"?

Protected method `typecast` uses in `phpTypecast()` method. Should we add notice and tests there also?

Reply:
Ok. Later

Reply:
Comment added.